packages/subgraph/src/mappings/resolver.ts

```diff
@@ -67,6 +67,8 @@ export function handleSet(event: Set): void {
             token.save();
         }
     }
+    resolverEntry.targetAddress = event.params.target;
```

Comment:
I don't understand why this has to be set here. Doesn't `getOrInitResolverEntry` already handle it?

Reply:
no - we don't set it in `getOrInitResolverEntry` because we have logic slightly above in the if block which relies on `targetAddress` not being set yet (for example removing a token from the resolver), which is why we do it at the end of the call.